Sources/EpoxyCollectionView/ViewControllers/CollectionViewController.swift

```diff
@@ -20,6 +20,24 @@ open class CollectionViewController: UIViewController {
     super.init(nibName: nil, bundle: nil)
   }
 
+  /// Initializes a collection view controller and configures its collection view with the provided
+  /// layout and sections once the view loads.
+  public convenience init(
+    layout: UICollectionViewLayout,
+    @SectionModelBuilder sections: () -> [SectionModel])
+  {
+    self.init(layout: layout, sections: sections())
+  }
+
+  /// Initializes a collection view controller and configures its collection view with the provided
+  /// layout and items once the view loads.
+  public convenience init(
+    layout: UICollectionViewLayout,
+    @ItemModelBuilder items: () -> [ItemModeling])
```

Comment:
🙌  this is really great, and makes me think we should add a `setItems()` function on `CollectionView` / `CollectionViewController` as well. I've been adding it in an extension in my own projects because I almost always only have a single section

Reply:
Added. in https://github.com/airbnb/epoxy-ios/pull/17/commits/c3a9425c744ff866dd2aa2e70ed65f42e32654fb